### PR TITLE
Config on Fargate: Try sending logs to Logit

### PR DESCRIPTION
**Pull request by Krenair**
This is experimental and may not work.

### CODE REVIEW

File: terraform/modules/hub/files/tasks/hub-config-fargate.json [hunk 1]
```diff
@@ -24,12 +42,12 @@
...
     "logConfiguration": {
-      "logDriver": "awslogs",
+      "logDriver": "awsfirelens",
       "options": {
-        "awslogs-group": "${deployment}-hub",
-        "awslogs-region": "${region}",
-        "awslogs-stream-prefix": "config-nginx",
-        "awslogs-create-group": "true"
+        "Name": "es",
+        "Host": "https://${logit_elasticsearch_url}:443",
```
Comment (adityapahuja) [inline terraform/modules/hub/files/tasks/hub-config-fargate.json:48]: Do we need `:443` if we use `https://`?

Comment (adityapahuja) [reply terraform/modules/hub/files/tasks/hub-config-fargate.json:48]: TLS support is currently disabled by default. Can we enable TLS support?

Comment (Krenair) [reply terraform/modules/hub/files/tasks/hub-config-fargate.json:48]: This is just the string copied directly from the journalbeat config. We probably can get rid of the port, but I think TLS support is already enabled?

Comment (adityapahuja) [reply terraform/modules/hub/files/tasks/hub-config-fargate.json:48]: By default, tls is set to off. Source: https://docs.fluentbit.io/manual/pipeline/outputs/tcp-and-tls

Comment (Krenair) [reply terraform/modules/hub/files/tasks/hub-config-fargate.json:48]: Hm okay yes, we need to get rid of https:// and :443 here and set that.

**Pull request merged**